index.js

```diff
@@ -107,8 +107,22 @@ let runID = new Date().getTime();
 
 function completer(line) {
   let completions = '/summarize /save /run /quit /explore /assert /undo /manual'.split(' ')
-  
-  completions.concat(tasks)
+  let filesAndFolders = fs.readdirSync('./')
+  filesAndFolders = filesAndFolders.map(file => {
+    let stats = fs.statSync(file)
+    if(stats.isDirectory()){
+      return file + '/' // add a '/' at the end of the folder name
+    } else {
+      return file
+    }
+  })
+
+  if (line.startsWith('/run ')){
+    var partialName = line.substring(5) // works only when there is a space after /run
+    var matches = filesAndFolders.filter(name => name.startsWith(partialName))
+    var fullCommand = matches.map(name => '/run ' + name)
```

Comment:
use `let` instead of `var`

Reply:
Oh ok got it. 
I did see one var [here](https://github.com/testdriverai/testdriverai/blob/3d510b1b227411d4379c61f3275c82252fdf1d44/index.js#L106)